packages/http-server/src/server.ts

```diff
@@ -14,13 +15,24 @@ export const createServer = <LoaderInput>(
   const server = fastify().register(fastifyAcceptsSerializer, {
     serializers: [
       {
-        regex: /json$/,
+        regex: /application\/\S*json$/,
```

Comment:
```
crapplication/json
application/mjson
application/vnd.xoxo.json
application/vnd.api+xml;q=json
```

Heads up that it will also match the above. Maybe a good candidate for the unit tests.

Reply:
I'm trying to see if I can replace this part and use is-type as well, the regex can fail and I do not feel exactly that comfortable.